alws/crud/repo_exporter.py

```diff
@@ -0,0 +1,143 @@
+    now = datetime.datetime.now()
+    query = select(models.RepoExporter.fs_exporter_href,
+                   models.RepoExporter.path,
+                   models.Repository.pulp_href).where(
+        models.RepoExporter.exported_id == export_id).join(
+        models.Repository).filter(
+        models.RepoExporter.repository_id == models.Repository.id)
+    async with db.begin():
+        await db.execute(
+            update(models.ExportTask).where(
+                models.ExportTask.id == export_id).values(exported_at=now,
+                                                          status=1))
+        response = await db.execute(query)
+        await db.commit()
+    exported_paths = []
+    for fs_exporter_href, fse_path, pulp_href in response:
+        latest_version_href = await pulp_client.get_repo_latest_version(
+            pulp_href)
+        fse_task = await pulp_client.export_to_filesystem(
+            fs_exporter_href, latest_version_href)
+        await pulp_client.wait_for_task(fse_task['task'])
+        exported_paths.append(fse_path)
+        await pulp_client.delete_filesystem_exporter(fs_exporter_href)
+    async with db.begin():
+        await db.execute(
+            update(models.ExportTask).where(
+                models.ExportTask.id == export_id).values(exported_at=now,
+                                                          status=3))
+        await db.commit()
+    return exported_paths
+
+
+async def create_filesystem_exporter(name : str, path: str) -> str:
+    pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
+                             settings.pulp_password)
+    result = await pulp_client.create_filesystem_exporter(name, path)
+    return result
+
+async def list_filesystem_exporters() -> list:
+    pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
+                             settings.pulp_password)
+    result = await pulp_client.list_filesystem_exporters()
+    return result
+
+async def get_filesystem_exporter(fse_pulp_href: str):
+    pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
+                             settings.pulp_password)
+    result = await pulp_client.get_filesystem_exporter(fse_pulp_href)
+    return result
+
+async def update_filesystem_exporter(fse_pulp_href: str,
+                                     fse_name: str,
+                                     fse_path : str):
+    pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
+                             settings.pulp_password)
+    result = await pulp_client.update_filesystem_exporter(
+        fse_pulp_href, fse_name, fse_path)
+    return result
+
+async def delete_filesystem_exporter(fse_pulp_href: str):
+    pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
+                             settings.pulp_password)
+    result = await pulp_client.delete_filesystem_exporter(
+        fse_pulp_href, fse_name, fse_path)
```

Comment:
`fse_name` and `fse_path` vars doesn't declare for function, looks like they are unnecessary

Reply:
Fixed. Thanks!